.mockery.yaml

```diff
@@ -262,11 +262,21 @@ packages:
       ORM:
       Runner:
       PipelineParamUnmarshaler:
-  github.com/smartcontractkit/chainlink/v2/core/services/promreporter:
+  github.com/smartcontractkit/chainlink/v2/core/services/headreporter:
     config:
-      dir: core/internal/mocks
+      dir: "{{ .InterfaceDir }}"
+      filename: "{{ .InterfaceName | snakecase }}_mock.go"
+      inpackage: true
+      mockname: "Mock{{ .InterfaceName | camelcase }}"
     interfaces:
+      HeadReporter:
       PrometheusBackend:
+  github.com/smartcontractkit/libocr/commontypes:
+    config:
+      dir: "common/types/mocks"
+      mockname: "Mock{{ .InterfaceName | camelcase }}"
```

Comment:
Typically we leave off the `Mock` prefix when using a `package mocks`
```suggestion
```

Reply:
done